test/transformer/create-mock-values.test.ts

```diff
@@ -6,9 +6,9 @@ describe('create-mock-values', () => {
   }
 
   interface Interface {
-    property: string;
-    method(): void;
-    subInterface: SubInterface;
+    property: string | null;
```

Comment:
Why do we need null?

Reply:
`property` (and the other properties) of `Interface` is assigned null in the `should create the mock merging the null values provided` test and they are asserted as such too.